documentation/topics/actions/actions.md

```diff
@@ -1,14 +1,14 @@
 # Actions
 
-In Ash, actions are the primary way to interact with your resources. All actions are run in a transaction if the data layer supports it. There are five types of actions:
+In Ash, actions are the primary way to interact with your resources. There are five types of actions:
 
 * [Read](documentation/topics/actions/read-actions.md)
 * [Create](documentation/topics/actions/create-actions.md)
 * [Update](documentation/topics/actions/update-actions.md)
 * [Destroy](documentation/topics/actions/destroy-actions.md)
 * [Generic](documentation/topics/actions/generic-actions.md)
 
-Each action has its own set of options, ways of calling it, and ways of customizing it. See the relevant guide for specifics on each action type. This topic focuses on idiomatic ways to use actions, and concepts that cross all action types.
+Out of above, Create, Update, Destroy actions are run in a transaction if the data layer supports it. Read and Generic actions are NOT run in a transaction. Each action has its own set of options, ways of calling it, and ways of customizing it. See the relevant guide for specifics on each action type. This topic focuses on idiomatic ways to use actions, and concepts that cross all action types. 
```

Comment:
Not to keep nitpicking, but they all *can* be run in a transaction. The only thing that is different is the default value for the `transaction?` option. So maybe something like. "All actions can be run in a transaction. Create, update and destroy actions are run in a transaction by default, whereas read and generic actions require opting in with `transaction? true` in the action definition". Maybe that was too wordy, but something like it could make sense.

Reply:
Clear wording solves problems. 

I'll make those changes.